pkgs/terraform-config-inspect/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "terraform-config-inspect";
+  version = "20210427";
+  owner = "davido-nw";
+  rev = "v${version}";
+
+  src = fetchFromGitHub {
+    inherit owner rev;
+    repo = pname;
+    sha256 = "1x9ifpk28r9hldpvig92zsvrl5hxzc0gj75q5xr20zb3jb22q11n";
+  };
+
+  vendorSha256 = "14r0q13hi8x7h6kkb26i8v1ialwk86ykgmn53i7874w1kvmzx2hg";
+
+  buildFlagsArray = [
+    "-ldflags="
+    "-s"
+    "-w"
+    "-X github.com/${owner}/${pname}/cmd.Version=${rev}"
+  ];
+
+  meta = with lib; {
+    description = "Generate an ssh_config from cloud resources";
+    homepage = "https://github.com/sarcasticadmin/sshcb";
+    maintainers = with maintainers; [ sarcasticadmin ];
```

Comment:
I would just keep this to say that its `NWI` like we did for: https://github.com/Nebulaworks/nix-garage/blob/master/pkgs/aws-key-rotator/default.nix#L21